include/tvm/ir_pass.h

```diff
@@ -326,6 +326,15 @@ Stmt RewriteUnsafeSelect(Stmt stmt);
  */
 Stmt LowerStorageAccessInfo(Stmt stmt);
 
+/*!
+ * \brief insert the mark of device for the hardware accelarator when 
```

Comment:
decorate the stmt with a device scope, this is helpful for hardware accelerator without thread blocks